scico/functional/_indicator.py

```diff
@@ -51,7 +51,7 @@ def prox(
         self, v: Union[JaxArray, BlockArray], lam: float = 1.0, **kwargs
     ) -> Union[JaxArray, BlockArray]:
         r"""Evaluate the scaled proximal operator of the indicator over
-            the non-negative orthant:
+            the non-negative orthant, :math:`I`,:
```

Comment:
On its own, perhaps "I" might be confused for the identity matrix? Perhaps write something like I_{[\mb{x} > 0]} to explicitly include the set of which it is the indicator?